src/Bridge/NelmioApiDoc/Parser/ApiPlatformParser.php

```diff
@@ -129,8 +133,8 @@ private function parseResource(ResourceMetadata $resourceMetadata, string $resou
     private function getGroupsForItemAndCollectionOperation(ResourceMetadata $resourceMetadata, string $operationName) : array
```

Comment:
Actually, the logic related to this whole function needs to be reviewed. I gave it a quick look and things look very fishy here.